01-davinci.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title: First Steps in LHCb
+subtitle: Introduction to the LHCb Software
+minutes: 10
+---
+
+> ## Learning Objectives {.objectives}
+> * Which basic problems is the LHCb software framework designed to solve?
+> * Learn the key concepts needed to work with the LHCb software
+>
+
+A HEP experiment like LHCb needs to solve many computational challenges, some of which are
+
+ - How do we collect data as it is recorded by the detector?
+ - How do we filter and process the recorded data efficiently?
+ - How do we manage all the complex tasks required to work with particle events?
+ - How do we organize all the data of a single particle event in a flexible way?
+ - How do we configure our software flexibly without having to recompile it?
+
+At LHCb, we base our software on the [Gaudi](https://proj-gaudi.web.cern.ch/proj-gaudi/) framework, which was specifically designed with the above questions in mind.
+In order to understand how the LHCb software works, you should know about the following basic Gaudi concepts:
+
+ - Because the individual collisions are almost completely independent of each other, it makes sense to process them one by one (as a *stream*), without holding them all in memory at once.
```

Comment:
This is not quite true. We assume that bunch crossings (a.k.a events) are independent. But collisions can actually be correlated (accidentally) if you assign a particle to the wrong PV.